test/e2e/persistent_volumes.go

```diff
@@ -522,6 +515,14 @@ var _ = framework.KubeDescribe("PersistentVolumes", func() {
 				nfsServerPod = startVolumeServer(c, NFSconfig)
 				serverIP = nfsServerPod.Status.PodIP
 				framework.Logf("NFS server IP address: %v", serverIP)
+			} else {
+				// If the nfs server does exist, check that it is Running and the volume is exported.
```

Comment:
Per our Fri conversation:  I'm going to open another PR to refactor the pattern we use for creating / destroying the NFS server pod.  We will instead create a new pod for each test spec within the test's namespace.  This will increase the robustness of each test since they will have a dedicated volume to test against.